$NetBSD: patch-CMakeLists.txt,v 1.2 2026/05/19 14:36:35 adam Exp $

We don't need another packaging system.
Don't make app bundle on Darwin.

--- CMakeLists.txt.orig	2026-04-17 18:27:01.841352595 +0000
+++ CMakeLists.txt
@@ -26,12 +26,12 @@ if(CMAKE_SYSTEM_NAME STREQUAL "Linux")
   set(LINUX ON)
 endif()
 
-if(LINUX)
+if(FALSE)
   include(cmake/Rpm.cmake)
   include(cmake/Deb.cmake)
 endif()
 
-if(APPLE)
+if(FALSE)
   include(cmake/Dmg.cmake)
 endif()
 
@@ -406,7 +406,7 @@ endif()
 
 add_executable(strawberry)
 
-if(APPLE)
+if(FALSE)
   set_target_properties(strawberry PROPERTIES MACOSX_BUNDLE TRUE)
   set_target_properties(strawberry PROPERTIES MACOSX_BUNDLE_INFO_PLIST "${CMAKE_CURRENT_BINARY_DIR}/dist/macos/Info.plist")
 endif()
@@ -1588,7 +1588,7 @@ endif()
 
 target_link_libraries(strawberry PUBLIC strawberry_lib)
 
-if(NOT APPLE)
+if(TRUE)
   install(TARGETS strawberry RUNTIME DESTINATION bin)
 endif()
 
